SPECS/kernel-mshv/config

```diff
@@ -1,6 +1,6 @@
 #
```

Comment:
Worth noting this config is not exactly the same as the one used in 2.0. This is due to some changes added in https://github.com/microsoft/azurelinux/commit/2d5057c38f249a0eac7685e68eb61e9d8b47fd0e, which I believe they are 3.0 specific. But the diff is identical to https://github.com/microsoft/azurelinux/pull/9261/files#diff-4571356362908e92951dedca8a59b37247916e8bbdc2834be1709d2b0a69b566. So the config signatures are different

Reply:
LGTM. Back then I was just syncing to the main branch.